shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -57,11 +57,11 @@
 </template>
 
 <template name="_loginButtonsLoggedOutDropdown">
-  <h4>Sign in</h4>
-
   {{#if isDemoUser}}
-    <p class="demo-note">Sign in to make your demo data permanent.</p>
+    <h4>Create Account</h4>
```

Comment:
In the screenshot, the 'a' seems to show up in lowercase. In this code, it's uppercase.

All I really hope is that this string is consistent with what's in the top-right.